Sources/Datadog/Logger.swift

```diff
@@ -17,6 +17,12 @@ public enum LogLevel: Int, Codable {
     case critical
 }
 
+public enum ErrorAttributes: String {
```

Comment:
Why making it `public`? The logs we create use many other attributes, but here we list only few of them. I'd rather keep it `internal`. If this is inspired by `public DDTags`, then it's not exactly the same, because tracing tags behave differently: we expect the user to give us those particular tags, but here, those error attributes are created internally.

Another thing: why do we need `enum`, instead of `struct`? I don't see how we leverage the enumeration aspect. Because of `enum`, we need to call `.rawValue` each time we reference `kind`, `message` or `stack` internally. Also, for other similar concepts (`DDTags`, `RUMContextIntegration.Attributes`, `LoggingWithActiveSpanIntegration.Attributes`) we use `struct`, so this `enum` is inconsistent.

Reply:
after the discussion at daily, we decided to keep them open/customizable